.github/workflows/publish-docker-latest.yml

```diff
@@ -1,17 +1,20 @@
 ---
 on:
-  push:
-    tags:
-      - 'v[0-9]+.[0-9]+.[0-9]+'
+  release:
+    types: [published]
 
 name: Publish latest image to Docker Hub
 
 jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v1
+      - uses: actions/checkout@v2
+      - name: Check if current release is latest
+        run: echo "##[set-output name=is_latest;]$(sh .github/is-latest-release.sh)"
+        id: release
       - name: Publish to Registry
+        if: steps.release.outputs.is_latest == 'true'
```

Comment:
I'm not sure `steps.release.outputs.is_latest` is what we want here, this is not based on the semver format.

Reply:
Yes, the script `is-latest-release.sh` check about the semver format. Why did you think it's not?

Reply:
I don't understand, where did you wrote this variable `steps.release.outputs.is_latest`?

Reply:
HO! this `echo "##[set-output name=is_latest;]$(sh .github/is-latest-release.sh)"` is the way you write into some variables in the github actions!? OK nice!

Reply:
I set the `is_latest` variable here:
```yml
        run: echo "##[set-output name=is_latest;]$(sh .github/is-latest-release.sh)"
        id: release
```